composeApp/src/androidMain/kotlin/id/andriawan/cofinance/auth/GoogleAuthManager.android.kt

```
@@ -0,0 +1,78 @@
+
+            val request = GetCredentialRequest.Builder()
+                .addCredentialOption(googleIdOption)
+                .build()
+
+            try {
+                val result = credentialManager.getCredential(
+                    context = context,
+                    request = request
+                )
+                handleSignInResult(result)
+            } catch (e: GetCredentialCancellationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)